scss/mixins/_container.scss

```diff
@@ -1,9 +1,10 @@
 // Container mixins
 
-@mixin make-container($gutter: $container-padding-x) {
+@mixin make-container($gutter: $grid-gutter-width) {
```

Comment:
Wouldn't this change the layout as we've defined `$container-padding-x` to be `$grid-gutter-width / 2`?

Reply:
Well spotted, the thing is it the variable wasn’t defined in line 4 so it was always default

Reply:
Good spot guys, adjusted to take /2 of the padding as it has been done in `make-row` mixin.
I didn't want to use container-padding-x so it would use the same variables to prevent mismatch of both values as per issue above